travelerproblem.py

```diff
@@ -108,18 +133,24 @@ def number_of_iterations(self, value: int):
 
     def carry_experiment(self) -> carry_exp_desc:

```

Comment:
This method looks a bit too big, maybe you would try to extract smaller parts from it?